IQSS-Dataverse-PullRequest.xml

```diff
@@ -4,39 +4,90 @@
   <description>Automated builds and testing of Pull Requests made against https://github.com/IQSS/dataverse/tree/develop</description>
   <keepDependencies>false</keepDependencies>
   <properties>
-    <com.sonyericsson.jenkins.plugins.bfa.model.ScannerJobProperty plugin="build-failure-analyzer@1.22.0">
+    <hudson.plugins.jira.JiraProjectProperty plugin="jira@3.0.10"/>
+    <com.sonyericsson.jenkins.plugins.bfa.model.ScannerJobProperty plugin="build-failure-analyzer@1.23.0">
       <doNotScan>false</doNotScan>
     </com.sonyericsson.jenkins.plugins.bfa.model.ScannerJobProperty>
     <com.coravy.hudson.plugins.github.GithubProjectProperty plugin="github@1.29.4">
-      <projectUrl>https://github.com/IQSS/dataverse/</projectUrl>
+      <projectUrl>https://github.com/OdumInstitute/dataverse/</projectUrl>
```

Comment:
@donsizemore can you please remind me if we want this to be "OdumInstitute" or "IQSS"?